lib/libtrakt/exceptions.py

```diff
@@ -1,8 +1,43 @@
-class traktException(Exception):
-    pass
+from requests.exceptions import (ConnectionError, Timeout, TooManyRedirects)
 
-class traktAuthException(traktException):
-    pass
 
-class traktServerBusy(traktException):
-    pass
+class TraktException(Exception):
```

Comment:
Exceptions should be called Errors (assuming they are an error) and not Exception.